ui/nginx.prod.conf

```diff
@@ -0,0 +1,30 @@
+# nginx server for static assets. For directive documentation, see
```

Comment:
This can stay here, but let's have just one .prod file naming convention instead of three (foo.bar.prod, foo.prod.bar, foo.bar-prod). I prefer nginx.conf.prod.

Reply:
Done